cves/CVE-2019-12308.yml

```diff
@@ -66,42 +80,26 @@ fixes_vcc_instructions: |
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
 - commit: afddabf8428ddc89a332f7a78d0d21eaf2b5a673
-  note:
-- commit: '09186a13d975de6d049f8b3e05484f66b01ece62'
-  note:
+  note: fix for Django version 2.2
+- commit: 09186a13d975de6d049f8b3e05484f66b01ece62
+  note: fix for Django version 2.1
 - commit: c238701859a52d584f349cce15d56c8e8137c52b
-  note:
+  note: fix for Django version 1.11
 vccs:
 - commit: b52c73008a9d67e9ddbb841872dc15cdd3d6ee01
-  note: This VCC was discovered automatically via archeogit.
-- commit: 46786b4193e04d398532bbfc3dcf63c03c1793cb
-  note: This VCC was discovered automatically via archeogit.
-- commit: c19b56f633e172b3c02094cbe12d28865ee57772
-  note: This VCC was discovered automatically via archeogit.
-- commit: 552abffab16cbdff571486b683e7e7ef12e46066
-  note: This VCC was discovered automatically via archeogit.
-- commit: '07988744b347302925bc6cc66511e34224db55ab'
-  note: This VCC was discovered automatically via archeogit.
-- commit: 16411b8400ad08f90c236bb2e18f65c655f903f8
-  note: This VCC was discovered automatically via archeogit.
-- commit: d13a9e44ded4e93570c6ba42ec84e45ddca2505b
-  note: This VCC was discovered automatically via archeogit.
-- commit: 5d199ba90113c47caf142e79151315ba2dc02b4b
-  note: This VCC was discovered automatically via archeogit.
-- commit: 2e8bc8f9862f127b0e69766c7fc7dc92f3156d20
-  note: This VCC was discovered automatically via archeogit.
-- commit: be67400b477c1b0e7e81766f41bbceed0de74bdc
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
+    Created the template-based widget rendering and failed to add check to url
+    template to validate url before using it to render a clickable url with html.
+    Template under django/contrib/admin/templates/admin/widgets/url.html
 - commit: ac2052ebc84c45709ab5f0f25e685bf656ce79bc
-  note: This VCC was discovered automatically via archeogit.
-- commit: cbe6d5568f4f5053ed7228ca3c3d0cce77cf9560
-  note: This VCC was discovered automatically via archeogit.
-- commit: 1c9ca83db2a73e8eb920fb50a2f1ee9a8228bcbf
-  note: This VCC was discovered automatically via archeogit.
-- commit: 4a103086d5c67fa4fcc53c106c9fdf644c742dd8
-  note: This VCC was discovered automatically via archeogit.
-- commit: 1c9233b1b9f903e4e2cb20a724e8c22aee4aacb2
-  note: This VCC was discovered automatically via archeogit.
+  note: |
+    This VCC was discovered automatically via archeogit.
+    Created the method to render the AdminURLFieldWidget using form text input,
+    and failed to make any validation check regarding the input being a properly
+    formatted url with no code attached to different parts of the url.
+    This commit also added unit tests for AdminURLWidgetTest which only covered
+    the more obvious cases and seemed to not be written with a security mindset
 upvotes_instructions: |
   For the first round, ignore this upvotes number.
```

Comment:
+3 Again, well written. This exploit wasn't as interesting though, could've been solved by just sanitizing all input from the start.